egs2/qasr_tts/RESULTS.MD

```diff
@@ -0,0 +1,17 @@
+- Environments
+  - python version: `3.8.12`
+  - espnet version: `espnet 0.10.7a1`
+  - chainer version: `chainer 6.0.0`
+  - pytorch version: `pytorch 1.10.0`
+
+- Model files 
+    - training config file (teacher model): `./conf/tuning/finetune_transformer.yaml`
+    - training config file: `./conf/tuning/train_conformer_fastspeech2.yaml`
+ 
+- Results 
+  - FastSpeech2 (Trans.) w/ PWG: (R = 1) 
+      - CER: 3.9
+      - WER: 9.13
+      - MOS
+        - intelligibility: 4.4 ± 0.06
```

Comment:
It would be great if you can reveal some details for the objective evaluation (also great to have some comparisons to other models/dataset) It would be not so informative by only providing the information with single dataset.

Reply:
I will be sharing the paper once its ready with all the trained models and comparisons.